docs/sql-ref-syntax-ddl-repair-table.md

```diff
@@ -39,6 +39,13 @@ MSCK REPAIR TABLE table_identifier
 
     **Syntax:** `[ database_name. ] table_name`
 
+* **`{ADD|DROP|SYNC} PARTITIONS`**
+
+    * If specified, `MSCK REPAIR TABLE` only adds partitions to the session catalog.
```

Comment:
typo: should be `If not specified`.

Reply:
I think it's better to put it in the end, and say `If not specified, ADD is the default.`

Reply:
Here is the PR https://github.com/apache/spark/pull/31633